dist/demo-app/README.md

```diff
@@ -1,74 +1,27 @@
 # Capital Framework
 
-A Capital Framework demo project. Use grunt to process the source files in `src/` and output them to `dist/`:
+A Capital Framework sample project.
+To preview it, run `python -m SimpleHTTPServer` and open http://localhost:8000 in a web browser.
```

Comment:
It *was* directly from the generator. But now that we only include the built assets in demo-app, I'm okay with it being different from the generator output. The only times we'll need to update it are when we introduce new components or make breaking changes and neither scenario is very common.